src/Domain/System/AlertLevelGateway.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/*
+Gibbon, Flexible & Open School System
+Copyright (C) 2010, Ross Parker
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+namespace Gibbon\Domain\System;
+
+use Gibbon\Domain\QueryableGateway;
+use Gibbon\Domain\Traits\TableAware;
+
+/**
+ * AlarmLevel Gateway.
+ *
+ * @version v25
+ * @since   v25
+ */
+class AlertLevelGateway extends QueryableGateway
+{
+    use TableAware;
+
+    private static $tableName = 'gibbonAlertLevel';
+    private static $primaryKey = 'gibbonAlertLevelID';
+
+    const LEVEL_HIGH = 1;
```

Comment:
I like the readability here that using constants adds 👍

Reply:
:-)